.github/workflows/release-exchange.yml

```
@@ -403,10 +429,10 @@ jobs:
           foreach ($path in $filePaths) {
             if (Test-Path $path -PathType Leaf) {
               Write-Host "Uploading $path"
-              aws s3 cp $path s3://${{ secrets.BASE_URL }}/exchanges/${{ steps.build.outputs.appVersion }}/${{ matrix.exchange.name }}/ --checksum-algorithm=CRC32 
+              aws s3 cp $path s3://${{ secrets.BASE_URL }}/exchanges/${{ matrix.exchange.id }}/${{ steps.build.outputs.appVersion }}/ --checksum-algorithm=CRC32 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)